sentinel/src/lib/history.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Utility to track duplicate keys in a certain period of time (periodMins).
+ * Removes expired keys after threshold is reached (purgeThreshold).
+ **/
+const moment = require('moment');
+
+let records = {};
+
+const now = () => {
+  return moment(new Date());
```

Comment:
Why not just `moment()` - I think it's the same. Then this function is probably no longer needed?